model/version.go

```diff
@@ -59,6 +59,9 @@ type Version struct {
 
 	// child patches will store the id of the parent patch
 	ParentPatchID string `bson:"parent_patch_id" json:"parent_patch_id,omitempty"`
+
+	// IsParent will store a boolean indicating wether the patch is a parent patch
+	IsParent bool `bson:"is_parent" json:"is_parent,omitempty"`
```

Comment:
You'll need to add an index for this field. Though is it needed? You can get the same info by checking if there are any versions whose values are in a version's ParentPatchID (if you do this the index will need to be on ParentPatchID instead)

Reply:
Why would this need an index? I don't think I query for versions based on this field at any point. 
Wouldn't iterating over all of our versions in the db be much more expensive than saving a value I already have access to when creating the version?

Reply:
Oops sorry I got way ahead of myself, the second sentence is what I really meant. This flag is only used in one place, but there you have the patch and can check that triggers.ChildPatches is not empty to see if it's a parent right?

Reply:
I don't have the patch available, I return if it's a parent before getting the patch.

Reply:
Right but you get it almost immediately. My point is that doing the extra query in that scenario is preferable to making the data model a bit more complex because the query is almost instantaneous

Reply:
I'd have to do that query even if I don't need the patch, and I'd be doing it for every notification. It feels like a lot, but I trust you:) 
done.